Modify requirements.md

lesser1 opened the pull request:
Remove IO patterns requ. after clearification
what was actually meant by that.
Add chaos testing requ.
Clearify description of label Ulm for external devs.

This had to be done due to feedback from supervisors and 
a more detailed answer / requ. from the examiner.

lesser1 committed:
Modify requirements.md:
Remove IO patterns requ. after clearification
what was actually meant by that.
Add chaos testing requ.
Clearify description of label Ulm for external devs.

nk-coding commented on docs/requirements/requirements.md:
```diff
@@ -24,7 +24,7 @@ This document is in the making since we are still in the requirements engineerin
   - Researcher *Toni*
 
-- External developers *Ulm*
+- *Ulm* -- a collective of developers of a similar system
```
I don't think that fits it too well.
They are rather doing performance / metric experiments and thus need to adapt an existing system

lesser1 replied:
How about "experienced users of similar systems; they do metric based performance experiments and regularly have to adapt the underlying reference architecture"?

nk-coding replied:
Sounds good to me

nk-coding commented on docs/requirements/requirements.md:
```diff
@@ -333,6 +342,13 @@ See also R-U-003.
+#### R-Conf-006: Configurable reliability
+
+- The system should be configurable regarding its services' reliability.
+- This would be "nice" for testing with chaos testing tools.
```
Doesn't the chaos toolkit simulate unreliable services by randomly killing them / doing other stuff?
Why does our system then need to be configurable too regarding service reliability (besides using the chaos toolkit)?

nk-coding replied:
or what do you exactly mean with "configurable reliability"?

lesser1 replied:
tbh Idk exactly how this toolkit works, I just took another look at the protocol with the examiner and it said "configureable reliability would be nice for tests; check if one can use toolkit Simian army, chaos toolkit". That's where I got this from. 

So we actually only need the requ. R-SArt-005 for chaos testing and I can discard R-Conf-006? Would be fine for me. I could add to R-SArt-005 something like "either the toolkit simulates unreliability or the system must enable this configuration itself"

nk-coding replied:
Yeah, merging into the other requirement sounds good

lesser1 committed:
Modify requirements.md:
Update stakeholder descriptions,
Clearify requ.,
Strike out elasticity policies to avoid confusion about
what exactly is the diff. between elasticity and scaling
